src/services/account/hooks/index.js

```diff
@@ -16,7 +17,7 @@ exports.before = {
 		auth.restrictToAuthenticated()
 	],
 	create: [
-		auth.hashPassword()
+		ifNotLocal(auth.hashPassword())
```

Comment:
Okay, so why do we even need this? I mean when you call accountService.create({..., password: 123, ...}) it should be fine. And creating an account without password should not be possible.

What kind of error do you get when calling it from the other service?